plugins/@grouparoo/postgres/src/lib/connect.ts

```diff
@@ -1,8 +1,14 @@
 import { Client } from "pg";
+import format from "pg-format";
 import { ConnectPluginAppMethod } from "@grouparoo/core";
 
 export const connect: ConnectPluginAppMethod = async ({ appOptions }) => {
   const client = new Client(appOptions);
   await client.connect();
+
+  if (appOptions.schema) {
+    await client.query(format(`SET search_path TO %L;`, appOptions.schema));
```

Comment:
does this change it for the "shared" connection. That might be a problem.

Reply:
We have a shared connection per App, and this option is per App.  Said another way, if you want to use 2 Schemas, you need 2 Apps.   This is the same security model as having a user/connection per-database.

Reply:
Score. Thanks.